src/main.js.tmp

```diff
@@ -0,0 +1,8 @@
+import Level from './level';
```

Comment:
Delete this .tmp file.  

You might add `*.tmp` to `.gitignore` to prevent this in the future.